src/gamekit.cs

```diff
@@ -1675,31 +1676,37 @@ interface GKAchievementDescription : NSSecureCoding {
 		bool Replayable { [Bind ("isReplayable")] get; }
 		
 #if MONOMAC
+		[NoiOS][NoMacCatalyst][NoWatch][NoTV]
```

Comment:
You can use the same `using` dance here to remove the `#if MONOMAC` condition.

Reply:
Looking.

Reply:
Fixed